Mathlib/CategoryTheory/Limits/IndYoneda.lean

```diff
@@ -201,12 +198,41 @@ lemma colimitCoyonedaHomIsoLimit_π_apply (f : colimit (D.rightOp ⋙ coyoneda)
 
 end ProCoyonedaContravariant
 
+section IndYonedaCovariant
+
+variable (D : Iᵒᵖ ⥤ Cᵒᵖ) (F : Cᵒᵖ ⥤ Type u₂)
```

Comment:
Should this be stated in terms of `D : I ⥤ C` instead?

Reply:
Good question. This actually applies to all four variants. They all could be stated so that the LHS "always applies" or that the RHS "always applies". Since the two existing variants are stated so that the RHS always applies, I did the same for the new variants. In your application, you need the RHS version, but in my application I actually need the LHS version. Of course, `D.unop.op` is definitionally equal to `D`, but relying on things like this is how you get goals that can only be solved by `erw`, so l guess I'll add four more variants. The hardest part will be the naming...

Reply:
I have added the variants.